src/core/sagas/autocomplete.ts

```diff
@@ -17,12 +19,13 @@ export namespace Tasks {
       const suggestionsRequest = effects.call(
         [flux.clients.sayt, flux.clients.sayt.autocomplete],
         query,
-        Selectors.autocompleteSuggestionsRequest(flux.config)
+        Requests.autocompleteSuggestions(flux.config)
       );
       const config = flux.config.autocomplete.recommendations;
+      // fall back to default mode "popular" if not provided
```

Comment:
This mentions what it's doing, but not why. I think @Vic-Dev meant explaining why we fall back to "popular" as opposed to the other modes.